cmdstanpy/stanfit.py

```diff
@@ -1330,3 +1337,113 @@ def save_csvfiles(self, dir: str = None) -> None:
         :param dir: directory path
         """
         self.runset.save_csvfiles(dir)
+
+
+def from_csv(dir: str = None) -> Union[CmdStanMCMC, CmdStanMLE, CmdStanVB]:
```

Comment:
It may be overly nitpicky, but I think including the method keyword as optional would be helpful in a statically typed context. It would be annoying to have to disambiguate the union type after every call if you know the method ahead of time. 

Other than the failing tests, this looks good to me

Reply:
happy to add some flexibility here.    changed logic, added checks and tests, also missing test files.  this time for sure?